app/controllers/sections/items/ItemAlcoholStrengthController.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.sections.items
+
+import controllers.actions._
+import forms.sections.items.ItemAlcoholStrengthFormProvider
+import models.requests.DataRequest
+import models.{Index, Mode}
+import navigation.ItemsNavigator
+import pages.sections.items.ItemAlcoholStrengthPage
+import play.api.data.Form
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import services.UserAnswersService
+import views.html.sections.items.ItemAlcoholStrengthView
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class ItemAlcoholStrengthController @Inject()(
+                                       override val messagesApi: MessagesApi,
+                                       override val userAnswersService: UserAnswersService,
+                                       override val navigator: ItemsNavigator,
+                                       override val auth: AuthAction,
+                                       override val getData: DataRetrievalAction,
+                                       override val requireData: DataRequiredAction,
+                                       override val userAllowList: UserAllowListAction,
+                                       formProvider: ItemAlcoholStrengthFormProvider,
+                                       val controllerComponents: MessagesControllerComponents,
+                                       view: ItemAlcoholStrengthView
+                                     ) extends BaseItemsNavigationController with AuthActionHelper {
+
+  def onPageLoad(ern: String, draftId: String, idx: Index, mode: Mode): Action[AnyContent] =
+    authorisedDataRequestAsync(ern, draftId) { implicit request =>
+      //TODO: Add back in when flow is properly enabled - commented out to allow JT to hit directly for now
```

Comment:
Does a particular page or a set of pages need to be built before this? I have a feeling it might get lost if we don't tag it with the page name e.g. ITM05

Reply:
I don't really want to say, because it depends what the first page of the ITEM section is. So I'll keep in generic and say that.